build/deploy_artifactory.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+rootDir=$(readlink -f "${scriptDir}/..")
```

Comment:
rootDir appears unused. (Cf CodeClimate)

Reply:
It is used now.

good point for the documentation you are pointing out, but it is global to all Ystia products so maybe its migration to a public place should be addressed in #260